dynamic-ssl/src/main/java/org/wildfly/security/dynamic/ssl/DynamicSslSocketFactory.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2020 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.security.dynamic.ssl;
+
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLSocketFactory;
+import java.io.IOException;
+import java.net.InetAddress;
+import java.net.Socket;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.UnknownHostException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * SSLSocketFactory that is being used by DynamicSSLContext.
+ *
+ * @author <a href="mailto:dvilkola@redhat.com">Diana Krepinska (Vilkolakova)</a>
+ */
+final class DynamicSslSocketFactory extends SSLSocketFactory {
```

Comment:
Very minor comment, looks like this class name should be `DynamicSSLSocketFactory` instead of `DynamicSslSocketFactory` to be consistent with the naming convention used in other classes.

Reply:
Fixed